core/domain/learner_group_domain_test.py

```diff
@@ -126,3 +124,11 @@ def test_validation(self) -> None:
                 ['subtopic_1', 'subtopic_2'],
                 ['story_1', 'story_2']),
             'Learner group facilitator cannot be invited to join the group.')
+
+        try:
+            # Test that valid learner group does not raise error.
+            learner_group = self.VALID_LEARNER_GROUP
+            learner_group.validate()
+        except Exception as error:
+            raise utils.ValidationError(
+                'Unexpected validation error.') from error
```

Comment:
You don't need try-except here. Across our tests, we consider it implicit that test code not in an `self.assertRaisesRegexp` block should raise no exceptions. If an exception does get raised, it will cause the test to fail like we'd expect

Reply:
Thanks, that sounds reasonable!